lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -40,18 +41,18 @@
 <div class="wrapper-banner wrapper-banner-user">
     <section class="banner banner-user">
         <div class="message message-block message-notice">
-            <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening | h}</h2>
+            <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
             <div class="wrapper-copy-and-actions">
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if facebook_share_enabled:
                       <button class="action action-share-facebook btn-inverse btn-small icon-only" id="action-share-facebook"
                          onclick="FaceBook.share({
-                          share_text: '${facebook_share_text | escapejs}',
-                          share_link: '${share_url}',
-                          picture_link: '${full_course_image_url}',
-                          description: '${_('Click the link to see my certificate.') | escapejs}'
+                          share_text: '${facebook_share_text | n, js_escaped_string}',
+                          share_link: '${share_url | n, js_escaped_string}',
+                          picture_link: '${full_course_image_url | n, js_escaped_string}',
+                          description: '${_('Click the link to see my certificate.') | n, js_escaped_string}'
                           });">
                           <span class="icon fa fa-facebook-official" aria-hidden="true"></span>
```

Comment:
There's some twitter button JS below this on lines 62-68 that you should also apply the `js_escaped_string` filter to.

Reply:
@amangano-edx I guess you mean only to JS code as the rest would be html escaped.